x/resource/types/params.go

```diff
@@ -1,146 +1,307 @@
 package types
 
 import (
+	"context"
 	"fmt"
 
 	sdkmath "cosmossdk.io/math"
+	util "github.com/cheqd/cheqd-node/util"
+	didtypes "github.com/cheqd/cheqd-node/x/did/types"
+	oracletypes "github.com/cheqd/cheqd-node/x/oracle/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// NewParams creates a new FeeParams object with specified parameters
-func NewParams(image, json, defaultFee sdk.Coin, burnFactor sdkmath.LegacyDec) FeeParams {
-	return FeeParams{
-		Image:      image,
-		Json:       json,
-		Default:    defaultFee,
-		BurnFactor: burnFactor,
-	}
-}
-
 // DefaultFeeParams returns default cheqd module tx fee parameters
 func DefaultFeeParams() *FeeParams {
 	return &FeeParams{
+		Image: []didtypes.FeeRange{
+			{
+				Denom:     BaseMinimalDenom,
+				MinAmount: sdkmath.NewInt(20000000000),
+				MaxAmount: util.PtrInt(30000000000),
+			},
+		},
+		Json: []didtypes.FeeRange{
+			{
+				Denom:     BaseMinimalDenom,
+				MinAmount: sdkmath.NewInt(3500000000),
+				MaxAmount: util.PtrInt(60000000000),
+			},
+		},
+		Default: []didtypes.FeeRange{
+			{
+				Denom:     BaseMinimalDenom,
+				MinAmount: sdkmath.NewInt(6000000000),
+				MaxAmount: util.PtrInt(20000000000),
+			},
+		},
```

Comment:
Similarly, we should add USD ranges to compliment CHEQ, as well as prioritise adding USD coverage in pricing tests, unit and integration.